k8s/base/config/common-application.yml

```diff
@@ -20,3 +20,7 @@ powsybl-ws:
   elasticsearch:
     index:
       prefix: ${powsybl-ws.environment:}
+
+# default values for loadflow providers "Hades2" or "OpenLoadFlow"
+loadflow:
+  default-provider: OpenLoadFlow
```

Comment:
Why defined here, and not in loadflow-server-application.yml ?
And maybe, we should keep loadflow-server-application.yml file, if we need to add specific properties in the future ...